lib/password.js

```diff
@@ -0,0 +1,22 @@
+"use strict";
+
+module.exports = function (config, crypto) {
+    /**
+     * Hashes the content passed, but if no content was passed
+     * just return false as there's nothing to do.
```

Comment:
What is the use case for passing this function no content?

I think I'd rather either not check or throw an Error.